app/controllers/users_controller.php

```diff
@@ -1022,6 +1023,7 @@ function delete($id, $courseId = null)
     function checkDuplicateName($courseId = null)
     {
         if (!$this->RequestHandler->isAjax()) {
+            $this->log('MMMMMM-checkDuplicateName::'.$puid);
```

Comment:
Is this for temporary debug? If yes, could you please remove it? or make the log message more meaningful if you want to keep it.